### PR TITLE
lldb.md: add page

### PR DESCRIPTION
**Pull request by kevinushey**
Add a page for the [LLDB](https://lldb.llvm.org/) debugger.

### CODE REVIEW

File: pages/common/lldb.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+`lldb {{executable}}`
+
+- Attach lldb to running process with id {{PID}}:
+
+`lldb -p {{PID}}`
+
+- Wait for process to launch and attach to it:
+
+`lldb -w -n {{name}}`
```
Comment (agnivade) [inline pages/common/lldb.md]: Lets change name to process_name to clarify things better. Also, a question - what if there are 2 processes with the same name. If I use the command without the `-w` flag, which process will it attach to ?

File: pages/common/lldb.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# lldb
+
+> The LLDB Debugger.
```
Comment (sbrl) [inline pages/common/lldb.md]: What does LLDB stand for?

Comment (agnivade) [inline pages/common/lldb.md]: low level debugger ? like llvm.

Comment (kevinushey) [inline pages/common/lldb.md]: I followed in the steps of `tldr gdb` here, which also just has

```
gdb

The GNU Debugger.
```

Not sure if there's a better way to describe it? The same title is used at https://lldb.llvm.org/ FWIW.

File: pages/common/lldb.md (hunk 1 above)
Comment (kevinushey) [inline pages/common/lldb.md]: In this case, neither of those processes -- `lldb` will instead wait for a _new_ process to be launched with name `{{name}}`, and then attach to that. (This is incredibly handy when you need to attach to a child process launched by some other process)

**Comment by kevinushey** [comment]
Addressed PR feedback (and force-pushed to keep PR history clean). Thanks for the review!

**Comment by agnivade** [comment]
>  (and force-pushed to keep PR history clean).

FYI, you need not do that. Adding commits linearly helps keep track of the PR context. And then while merging, we have an option to squash all the commits together. So, that can be achieved while merging the PR. :smile:

File: pages/common/lldb.md (hunk 1 above)
Comment (agnivade) [inline pages/common/lldb.md]: Got it. In that case, lets change the description to -
`Wait for a new process to launch with the given name and attach to it:`

File: pages/common/lldb.md (hunk 2 above)
Comment (agnivade) [inline pages/common/lldb.md]: You can write the expanded version in brackets - LLDB(Low Level Debugger). That is a norm we follow in various pages.

**Comment by kevinushey** [comment]
> FYI, you need not do that. Adding commits linearly helps keep track of the PR context. And then while merging, we have an option to squash all the commits together. So, that can be achieved while merging the PR. 😄

Ah, I didn't realize that!

Now updated to address PR comments -- thanks!

**Pull request merged**